tests/singlerod/short/nekrs/rod_short.par

```diff
@@ -0,0 +1,25 @@
+[OCCA]
+backend = SERIAL
+deviceNumber = LOCAL-RANK
+
+[GENERAL] 
+#startFrom = rod_short.run1
```

Comment:
Yes, I don't have a restart file for this case right now.  It will be different from the restart file from the Nek5000 shortrod.  

```suggestion
```